eslint/index.js

```diff
@@ -0,0 +1,57 @@
+module.exports = {
+  rules: {
+    'proptype-definition': {
+      create(context) {
+        return {
+          AssignmentExpression(node) {
+            // helper func which targets possible react components
+            const findDeclaration = singleNode => {
```

Comment:
Can you extract this to the top of the file? No need to add the comment -maybe just change the name to indicate that it always returns a boolean! Something like: `hasDeclaration`

Either way, it will just read cleaner if it's separated from the actual rule.

Reply:
Got it.